helix-core/src/main/java/org/apache/helix/PropertyPathBuilder.java

```diff
@@ -105,6 +105,9 @@
     addEntry(PropertyType.CUSTOMIZEDVIEW, 2, "/{clusterName}/CUSTOMIZEDVIEW/{customizedStateType}");
     addEntry(PropertyType.CUSTOMIZEDVIEW, 3, "/{clusterName}/CUSTOMIZEDVIEW/{customizedStateType}/{resourceName}");
     addEntry(PropertyType.STATUS, 1, "/{clusterName}/STATUS");
+    addEntry(PropertyType.STATUS, 2, "/{clusterName}/STATUS/{scope}");
+    addEntry(PropertyType.STATUS, 3, "/{clusterName}/STATUS/{scope}/{scopeKey}");
```

Comment:
What are the other possible values regarding scope? I thought it would be a single node for the whole cluster status. But please share the detailed design here.

Reply:
I agree. Do we need more than one node?

Reply:
Do you have a concrete example? It seems to be over-engineering.
I think the entities in Helix world include,
Resource - we have current state nodes and external view nodes,
Instance - we have liveInstance nodes
Controller - we have controller nodes

And these STATUS nodes is to provide some level of aggregation. There is no other entity that we can or need to add here.

Reply:
Synced offline with @jiajunwang We decided to use `/{clusterName}/STATUS/{clusterName}`.  Currently there is no need to add the `{scope}` layer.